lib/screens/sign_in.dart

```diff
@@ -1,52 +1,30 @@
 import 'package:flutter/material.dart';
-import 'package:splitwise/Utils/assets.dart';
+import 'package:flutter_mobx/flutter_mobx.dart';
+import 'package:provider/provider.dart';
 
+import '../Utils/assets.dart';
 import '../common_methods/clip_react.dart';
 import '../common_methods/sized_box_widget.dart';
 import '../common_methods/text_formfield_widget.dart';
-import '../routes/navigation_functions.dart';
+import '../routes/navigator_service.dart';
 import '../routes/routes.dart';
+import '../store/authentication_store/sign_in_store.dart';
 import '../utils/colors.dart';
 import '../utils/common_strings.dart';
 
-class LoginPage extends StatefulWidget {
+class LoginPage extends StatelessWidget {
   const LoginPage({Key? key}) : super(key: key);
 
   @override
-  State<LoginPage> createState() => _LoginPageState();
-}
-
-class _LoginPageState extends State<LoginPage> {
-  bool changeState = false;
-  String value = 'jay';
-
-  final _formKey = GlobalKey<FormState>();
-
-  pushAndRemoveSignUp() {
-    Navigator.of(context).pushNamedAndRemoveUntil(
-      Routes.signupPage,
-      (routes) => false,
-    );
-  }
-
-  void _clickForSignIn() {
-    if (_formKey.currentState!.validate()) {
-      Navigator.pop(context);
-      Navigator.pushNamed(context, Routes.loginPage);
-    }
-  }
+  Widget build(BuildContext context) {
+    final readStore = context.read<SignInStore>();
 
-  void _changedValue(bool? value) {
-    setState(() {
-      changeState = !changeState;
-    });
-  }
+    final theme = Theme.of(context);
```

Comment:
If we are only going to use textTheme from the Theme.of then put it like this Theme.of(context).textTheme